### ISSUE TITLE
Capitalise "cc-by-sa-4.0" and friends?

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/tree-build#86]
It looks like the existing images have an images_by_ott `licence` field like:

```
CC-BY-SA 3.0 (http://creativecommons.org/licenses/by-sa/3.0/)
CC-BY 2.0 (http://creativecommons.org/licenses/by/2.0/)
```

And so on, whereas the get_wiki_images scripts saves this field as

```
cc-by-sa-3.0 (http://creativecommons.org/licenses/by-sa/3.0/)
cc-by-2.0 (http://creativecommons.org/licenses/by/2.0/)
```

(note also the last of hyphen before the version number)

It would be nice (but not essential) for this to be consistent. I. slightly prefer the caps style, as these are acronyms, so it might be more obvious to a casual user.

**Comment by davidebbo** [comment]
As it stands, I kept the string exactly as it appears in Wikimedia, but we could canonicalize it.

It would also be easy to fix up the DB after the fact with an UPDATE, e.g.

```sql
UPDATE OneZoom.images_by_ott
SET licence='CC-BY-SA 3.0 (https://creativecommons.org/licenses/by-sa/3.0)'
WHERE licence='cc-by-sa-3.0 (https://creativecommons.org/licenses/by-sa/3.0)';
```

**Comment by hyanwong** [comment]
Right, although it might be a pain to go through all the permutations of 2.0, 2.5, 3.0, 4.0 etc. We could probably split and capitalise it before writing it the first time? Perhaps a Python "canonicalise_cc_strings" function would be helpful?

**Comment by davidebbo** [comment]
Yes, definitely best to do it from the get go. Just pointing out that we can easily fix things up if we want to adjust things after the fact.

**Comment by hyanwong** [comment]
This might be a good candidate for "test-driven" development. Write the test first, make sure it fails, then fixup the code to get it passing.
